README.rst

```diff
@@ -1,17 +1,21 @@
-Pure Python RIS files parser
-============================
+Pure Python RIS files parser and reader
+=======================================
+
+Before, this project was named 'RISparser'.
```

Comment:
move this link to `HISTORY.rst`; indicate the api renaming for breaking changes

Reply:
Not completely sure what to do with this one. Added the changelog to HISTORY.rst.